AP-protocol.md

```diff
@@ -217,41 +219,13 @@ The flood can terminate when:
 
 Clients and servers operate with high level `Message`s which are disassembled into atomically sized packets that are routed through the drone network. The Client-Server Protocol standardizes and regulates the format of these messages and their exchange.
 
-The previously mentioned packets can be: Fragment, Ack, Nack, FloodRequest, FloodResponse.
+The previously mentioned packets can be: `Fragment`, `Ack`, `Nack`, `FloodRequest`, `FloodResponse`.
 
 As described in the main document, `Message`s must be serialized and can be possibly fragmented, and the `Fragment`s can be possibly dropped by drones.
 
-### Message
-
-`Message` is subject to fragmentation: see the dedicated section.
-
-`Fragment` (and `Fragment` only) can be dropped by drones.
-
-```rust
-#[derive(Debug)]
-pub enum ServerType {
-	ChatServer,
-	TextServer,
-	MediaServer,
-}
-
-#[derive(Debug)]
-pub struct Message {
-	message_data: MessageData,
-	routing_header: SourceRoutingHeader
-}
-
-#[derive(Debug)]
-// Part to be fragmented
-pub struct MessageData {
-	session_id: u64,
-	content: MessageContent
-}
-```
-
 ### Ack
 
-If a drone receives a Message and can forward it to the next hop, it also sends an Ack to the client.
+This Packet is sent by a client/server after receiving a Packet.
```

Comment:
Here should be 'Ack' instead of the First 'Packet'

Reply:
Solved